tailwind.config.js

```diff
@@ -19,8 +19,6 @@ module.exports = {
       fontFamily: {
         inter: ["inter"],
       },
-      // stretch the grid so that footer is always at the bottom
-      // even on pages with little content
```

Comment:
Why was this comment removed? 🤔

Reply:
@saidam90 if you think this comment should be removed, please open a new PR with a reason why it should be removed :)